### ISSUE TITLE
How to run phpunit inside container

### ISSUE THREAD

**Issue by bocharsky-bw** [pilot/symfony-docker-deploy#2]
@pilot Here you run tests directly on CircleCI machine, i.e. you should provision it first for a more complex application. But do you have any examples how to run PHPUnit inside the container? Isn't it a more robust way to test Docker applications - test them inside built containers to use *exactly* the same environment?

**Comment by pilot** [comment]
This is not correct to do inside container, because usual project counts a lot of them, which a linked between each others, so test can't be bypass without knowing about related containers.

**Comment by bocharsky-bw** [comment]
Hm, for unit tests it makes sense. But what about integration tests? Or Behat tests, when the system should work as a single unit?
